### PR TITLE
Adds direction signs around the station [V2]

### LINKED ISSUE #13764: Escape sign hidden by other two signs
**Issue Description**:
There is a useless "Escape" sign to the left of the bar entrance.

**What did you expect to happen**: 
A sign that's not visible without knowing its there defeats its purpose.

**What happened instead**:
The sign is hidden under the others.

**Why is this bad/What are the consequences**:
Not much outside of it being useless.

**Steps to reproduce the problem**:
Join the server and go to the place where sign is. (central primary hallway, left of the bar on the wall)

**When did the problem start happening**:

**Extra information**:
https://imgur.com/a/uqzWvNR

### PR DESCRIPTION
**Pull request by SabreML**
## What Does This PR Do

**[Re-opening of #13924, updated for new map changes.]**
Adds some new Direction Signs around the station.
Fixes #13764

## Why It's Good For The Game

Direction signs can help new players to get around, and this adds a direction sign to every department.
Also adds to the """realism""" of the station a bit, in my opinion.

## Images of changes

NSS Cyberiad 2021-02-09 171926
NSS Cyberiad 2021-02-09 171934
NSS Cyberiad 2021-02-09 171939
NSS Cyberiad 2021-02-09 171943
NSS Cyberiad 2021-02-09 171956
NSS Cyberiad 2021-02-09 172005

## Changelog
:cl:
add: Added new direction signs in the Port, Starboard and Central hallways.
tweak: Moved an APC and Status Display slightly to make room.
tweak: Renamed all direction signs to just 'direction sign', to avoid possible confusion.
/:cl:

### CODE REVIEW

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 1]
```diff
@@ -29763,6 +29758,12 @@
 	},
 /area/ai_monitored/storage/eva)
 "biO" = (
+/obj/structure/cable{
+	d1 = 1;
+	d2 = 2;
+	icon_state = "1-2";
+	tag = ""
```
Comment (dearmochi) [inline _maps/map_files/cyberiad/cyberiad.dmm:29765]:
```suggestion
	icon_state = "1-2"
```

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 2]
```diff
@@ -31986,6 +31973,12 @@
 /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers{
 	dir = 4
 	},
+/obj/structure/cable{
+	d1 = 1;
+	d2 = 2;
+	icon_state = "1-2";
+	tag = ""
```
Comment (dearmochi) [inline _maps/map_files/cyberiad/cyberiad.dmm:31980]:
```suggestion
	icon_state = "1-2"
```

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 3]
```diff
@@ -33088,6 +33081,12 @@
 /obj/machinery/atmospherics/unary/vent_pump/on{
 	dir = 1
 	},
+/obj/structure/cable{
+	d1 = 1;
+	d2 = 8;
+	icon_state = "1-8";
+	tag = ""
```
Comment (dearmochi) [inline _maps/map_files/cyberiad/cyberiad.dmm:33088]:
```suggestion
	icon_state = "1-8"
```

File: _maps/map_files/cyberiad/cyberiad.dmm [hunk 4]
```diff
@@ -90504,6 +90516,15 @@
 	},
 /turf/simulated/floor/plasteel,
 /area/crew_quarters/dorms)
+"sVH" = (
+/obj/machinery/status_display{
+	density = 0;
```
Comment (dearmochi) [inline _maps/map_files/cyberiad/cyberiad.dmm:90521]:
```suggestion
```

**Review by dearmochi — CHANGES_REQUESTED** [review]
A few things

File: _maps/map_files/cyberiad/cyberiad.dmm (hunk 1 above)
Comment (SabreML) [reply _maps/map_files/cyberiad/cyberiad.dmm:29765]: All of these were there already but yeah, why not remove them.

**Review by dearmochi — APPROVED** [review]

**Review by SteelSlayer — APPROVED** [review]

**Review by Kyep — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**